ios/RCTTWVideoModule.m

```diff
@@ -101,6 +101,14 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   }
 }
 
+RCT_EXPORT_METHOD(setRemoteAudioPlayback:(NSString *)participantSid enabled:(BOOL)enabled) {
+    TVIRemoteParticipant *participant = [self.room getRemoteParticipantWithSid:participantSid];
+    NSArray<TVIRemoteAudioTrackPublication *> *trackPublications = participant.remoteAudioTracks;
```

Comment:
Mind adding a check for a null participant?

Reply:
I just figure you might have a race condition where the track is added, and then they leave the room but in the meantime on the JS side someone is calling setRemoteAudioPlayback via the callback.